docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -234,7 +224,7 @@ For Python API
 
 
 ### Performance Hints
```

Comment:
do we need such big section about performance hints? I don't see anything specific for AUTO here. Can we just refer to common document about per hints written by @myshevts  ?